examples/next/faustwp-block-support/styles/_base.scss

```diff
@@ -0,0 +1,59 @@
+// Base styles
+
```

Comment:
Ideally we don't want the base styles since it will interfere with the globalStyleSheet. So you might find the the styles are not in parity with the editor.